scripts/spot-grid.py

```diff
@@ -135,10 +140,10 @@ def update_view(self):
         text_surface = cairo.ImageSurface(cairo.FORMAT_ARGB32, width, height)
         info = [
             "rotation: {:.1f} deg".format(numpy.rad2deg(self.rotation)),
-            "pitch-x: {:.2f} um".format(PIXEL_SIZE_UM * self.scale[0]),
-            "pitch-y: {:.2f} um".format(PIXEL_SIZE_UM * self.scale[1]),
-            "translation-x: {:.1f} um".format(PIXEL_SIZE_UM * self.translation[0]),
-            "translation-y: {:.1f} um".format(PIXEL_SIZE_UM * self.translation[1]),
+            "pitch-x: {:.2f} um".format(self.pixelsize * self.scale[0]),
+            "pitch-y: {:.2f} um".format(self.pixelsize * self.scale[1]),
+            "translation-x: {:.1f} um".format(self.pixelsize * self.translation[0]),
+            "translation-y: {:.1f} um".format(self.pixelsize * self.translation[1]),
```

Comment:
We do want to display these values in micrometers, I think this is now also in meters right?